composer.json

```diff
@@ -12,7 +12,7 @@
     "homepage": "https://github.com/ericpridham/flow",
     "keywords": ["Laravel", "Flow"],
     "require": {
-        "illuminate/support": "~8|~9",
+        "illuminate/support": "^8|^9|^10|^11",
```

Comment:
`~8`, `^8`, and `^8.0` are going to result in the same restrictions, so they are interchangeable. My choice to use the `^8` format throughout all these PRs was just personal preference. ie, I didn't convert `~8` to `^8` here for any functional reason.